fast_llm/layers/language_model/head.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         hidden_dim = self._tensor_space.get_tensor_dim(TransformerDimNames.hidden)
 
         self._loss_name = LanguageModelLossNames.multi_token_prediction_loss(prediction_distance)
+        self._logits_name = "logits" if prediction_distance == 0 else f"logits_{prediction_distance}"
```

Comment:
Used only once, could be inline. Do we plan on combining knowledge distillation with MTP?

Reply:
There is no short-term plan to combine knowledge distillation with MTP. Is there an issue supporting both though?

Reply:
I don't think it will be hard, but we need to ensure the number of prediction heads is compatible (teacher>=student?)